src/loaders/omeZarrReader.js

```diff
@@ -10,9 +10,12 @@ async function getJson(path, key) {
   return json;
 }
 
-// Mostly a javascript implementation of ome-zarr-py
-// https://github.com/ome/ome-zarr-py/blob/master/ome_zarr.py
-
+/**
+ * This class attempts to be a javascript implementation of ome-zarr-py
+ * https://github.com/ome/ome-zarr-py/blob/master/ome_zarr.py
+ * @param {String} zarrPath url to root zarr store
+ * @param {Object} rootAttrs metadata for zarr array
+ * */
```

Comment:
Can you add this class to the `documentations.yml` and make sure it looks right when you build?

Reply:
Thanks! Just added and built the docs -- can confirm things look correct.